core/templates/pages/exploration-editor-page/editor-navigation/editor-navigation.component.html

```diff
@@ -241,10 +241,28 @@
     width: 100%;
     z-index: 1040;
   }
+  .show-mobile-navbar-icon {
+    background-color: #e6e6e6;
+    border-left: 1.5px solid #000;
+    border-radius: 10px 0;
+    border-top: 1.5px solid #000;
+    bottom: 0;
+    color: #000;
+    font-size: 25px;
```

Comment:
Why is there not a `display: none` similar to the `navbar-mobile-container` class? Can you please attach a video of the PC view to show that the changes are correct?

Reply:
I forgot to add it when I removed the duplicated selector. It works now.

Reply:
I don't see a video showing that it looks correct on PC, can you please attach it?

Reply:
https://github.com/oppia/oppia/assets/125552595/90e1ce80-2c39-458c-b9b1-73e69fcd0150

Reply:
https://github.com/oppia/oppia/assets/125552595/49c2db0f-22e9-4602-a96b-86259a1144e3

Reply:
Thanks!